x-pack/plugins/uptime/public/apps/plugin.ts

```diff
@@ -132,6 +138,22 @@ export class UptimePlugin
         plugins.triggersActionsUi.alertTypeRegistry.register(alertInitializer);
       }
     });
+
+    if (plugins.fleet) {
+      const { registerExtension } = plugins.fleet;
+
+      registerExtension({
+        package: 'synthetics',
+        view: 'package-policy-create',
+        component: LazySyntheticsPolicyCreateExtension,
+      });
+
+      registerExtension({
+        package: 'synthetics',
+        view: 'package-policy-edit',
```

Comment:
do we also need something for p`ackage-policy-delete`? or it's covered in edit?

Reply:
These extensions only apply to particular views. There isn't a view for delete. Rather, you delete the policy simply in the existing fleet UI within the policies page.

Reply:
but we will need that when we want to delete monitors directly from uptime UI